charts/gatekeeper-artifacts/templates/config.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.dataSync }}
+apiVersion: config.gatekeeper.sh/v1alpha1
+kind: Config
+metadata:
+  name: config
+  namespace: "gatekeeper-system"
+spec:
+  sync:
+    syncOnly:
+      - group: ""
+        version: "v1"
+        kind: "Namespace"
+      - group: ""
+        version: "v1"
+        kind: "Pod"
```

Comment:
Before `syncOnly` was configurable to be able to choose the kinds. What does this syncing give us? Does it need to be configurable? Why is it possible to turn this crudely on or off?

Reply:
hmm, yes adjusting the `sync resource kinds` to be configurable does sound like an improvement.  
The entire `sync config` resource needs to be turned on/off as well. It is optional and we currently do not use any policy which requires kubernetes inventory access.

Reply:
Is this meant for audit? So we can see what and who were causing violation? Makes me wonder what else we can let OPA detect that violates the api requirements. Could it detect spamming? Could it wait longer responding on each violation coming from the same IP?